### PR TITLE
feat: added a golang resource

### LINKED ISSUE #484: [Add] Golang series by hitesh choudhary
### URL

https://youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa

### Description of the above provided site

This playlist contains the Golang series along with two mini projects

### Category

Youtube

### Sub Category

"Backend","APIs"

### PR DESCRIPTION
**Pull request by himanshu1221**
## Fixes Issue

closes #484 

## Changes proposed

- Added resources for Golang series

## Screenshots
Screenshot from 2023-04-27 11-45-50

## Note to reviewers

hey! thanks for reviewing my PR do let me know if there is any change required

### CODE REVIEW

File: database/languages/golang.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "name": "Hitesh Choudhary",
+    "description": "This Playlist of Hitesh Choudhary contains all the cocepts of Golang along with two mini projects",
```
Comment (CBID2) [inline database/languages/golang.json:4]:
```suggestion
    "description": "This playlist contains all the concepts of Golang along with two mini projects",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @himanshu1221! 😄 I just made some suggestions on how you can improve your description. Other than that, great job

File: database/languages/golang.json (hunk 1 above)
Comment (himanshu1221) [reply database/languages/golang.json]: hey @CBID2 i have applied the suggested changes

File: database/languages/golang.json [hunk 2]
```diff
@@ -1,3 +1,9 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "name": "Hitesh Choudhary",
+    "description": "This playlist contains all the concepts of Golang along with two mini projects",
+    "url": "https://youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa",
+    "category": "Other",
```
Comment (rupali-codes) [inline database/languages/golang.json:6]:
```suggestion
    "category": "youtube",
```

**Review by CBID2 — APPROVED** [review]
Looks great @himanshu1221! :) LGTM! :)

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**